snowshu/adapters/snowflake_common.py

```diff
@@ -54,4 +63,6 @@ def _build_conn_string(self) -> str:
         ]
         get_string = "?" + "&".join(get_args) if get_args else ""
 
+        logger.info(f"#### Connection String #### {base_conn}{schema}{get_string}")
```

Comment:
I believe this was added for debugging and is not necessary anymore. Have to be removed since is outputs sensitive info(credentials)

Reply:
definitely, i completly missed it ;<